packages/orchestration/test/examples/unbond.contract.test.ts

```diff
@@ -14,11 +20,42 @@ type StartFn =
 
 test('start', async t => {
   const {
-    bootstrap: { vowTools: vt },
+    bootstrap: { timer, vowTools: vt },
     brands: { ist },
     commonPrivateArgs,
+    mocks: { ibcBridge },
   } = await commonSetup(t);
 
+  const buildMocks = () => {
+    const makeDelegationsResponse = () =>
+      buildQueryResponseString(QueryDelegatorDelegationsResponse, {
+        delegationResponses: [
+          {
+            delegation: {
+              delegatorAddress: 'cosmos1test',
+              validatorAddress: 'cosmosvaloper1xyz',
+              shares: '1000000',
+            },
+            balance: { denom: 'uatom', amount: '1000000' },
+          },
+        ],
+        pagination: { nextKey: new Uint8Array(), total: 1n },
```

Comment:
Do we handle/support pagination?  Not sure it's worth the cost atm, but curious what it looks like. Maybe it's as simple as exposing a `nextKey` as an optional param in `getDelegations()` and the other interfaces that support it.